nextcord/application_command.py

```diff
@@ -420,11 +471,47 @@ def __init__(
         self.options: Dict[str, CommandOption] = {}
         self.children: Dict[str, ApplicationSubcommand] = {}
 
+        self.on_error: ApplicationErrorCallback = MISSING
```

Comment:
Sholdnt this default to None? Also this typehint is wrong?

Reply:
Why should it default to None? I don't really see any problems with the typehint either...

Reply:
i suppose because MISSING is only used when none means something, the typehint implies its never not `ApplicationErrorCallback` - of which, it may be `MISSING` in this snippet